.changeset/mean-trainers-tickle.md

```diff
@@ -0,0 +1,54 @@
+---
+'@clerk/clerk-js': minor
+'@clerk/clerk-expo': minor
+'@clerk/shared': minor
+'@clerk/types': minor
+---
```

Comment:
We should split up the changes into multiple changesets. If we publish it like that, every package will have the long text below even though it should only end up in `clerk-expo`, right?

Reply:
@LekoArts That's a correct suggestion! Thanks!